### ISSUE TITLE
"Could not establish connection. Receiving end does not exist" and Freeze/Unresponsive after entering decryption password.

### ISSUE THREAD

**Issue by error420** [marius-wieschollek/passwords-webextension#232]
### System Information
- Extension Version: 2.2.4.4461
- App Version: 2022.7.20-build4507
- Browser and Version: Chrome  104.0.5112.81
- Client OS and Version: Windows 11 22000.795 and Ubuntu 18

### Steps to reproduce
1. Open browser
2. Left click passwords extension icon
3. Enter decryption password and press submit "login" button

### Expected result
What should happen? Search box should appear and allow me to search for passwords

### Actual result
What does happen? The search box freezes and does not allow you to click anywhere else within the extension. You can retrieve bug reports if you do not log in and right click upon browser launch to access extension options. Clicking away from the popup will cause the extension to never open again until you exit chrome and then relaunch without entering decryption credentials.

### Browser log
<details>
<summary>Browser log</summary>

In Chrome
Error Log
[2022-08-08 18:55:12] Could not establish connection. Receiving end does not exist.
[2022-08-08 18:55:12] Could not establish connection. Receiving end does not exist.
[2022-08-08 18:55:12] Could not establish connection. Receiving end does not exist.

{
  "details": {
    "data": {
      "id": "***omitted***",
      "tab": null,
      "type": "queue.fetch",
      "reply": null,
      "silent": false,
      "sender": "background",
      "channel": "runtime",
      "payload": {
        "name": "mining"
      },
      "receiver": "background"
    },
    "message": "Could not establish connection. Receiving end does not exist.",
    "stack": "Error: Could not establish connection. Receiving end does not exist.\n    at I (chrome-extension://mhajlicjhgoofheldnmollgbgjheenbi/js/background.js:2:742452)",
    "time": 1659995712376
  },
  "error": {
    "name": "Error",
    "stack": "Error: Could not establish connection. Receiving end does not exist.\n    at I (chrome-extension://mhajlicjhgoofheldnmollgbgjheenbi/js/background.js:2:742452)",
    "string": "Error: Could not establish connection. Receiving end does not exist.",
    "message": "Could not establish connection. Receiving end does not exist."
  }
}

</details>

**Comment by Denter-** [comment]
Same problem here.

Activated logging for extensions, this is what I see:
Constantly repeated message
`
{"documentId":"12BB03476D5CB4234BA8DF8FE7B924F0","documentLifecycle":"active","frameId":27,"frameType":"sub_frame","fromCache":false,"initiator":"https://chat.google.com","ip":"142.251.39.14","method":"POST","parentDocumentId":"006BE5704D1E1525B5BA002323C2B1FF","parentFrameId":0,"requestId":"1550","statusCode":200,"statusLine":"HTTP/1.1 200","tabId":1779241132,"timeStamp":1660746308904.542,"type":"xmlhttprequest","url":"https://chat.google.com/u/0/webchannel/events?VER=8&SID=xiIEqXIXB5MkyvhwiLskfg&RID=91529&AID=4&zx=4emgnhgme03h&t=1"}`

**Comment by kernjuri** [comment]
the same problem on many windows pc and chrome browser here too,

**What does happen? The search box freezes and does not allow you to click anywhere else within the extension. You can retrieve bug reports if you do not log in and right click upon browser launch to access extension options. Clicking away from the popup will cause the extension to never open again until you exit chrome and then relaunch without entering decryption credentials.**

**Comment by marius-wieschollek** [comment]
Duplicate of https://github.com/marius-wieschollek/passwords-webextension/issues/230
